CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 0.15.0
+
+- Parsing the spec file by RPM is now performed only if really necessary, greatly improving performance in certain scenarios. (#212)
+- Checked that license is a valid SPDX license.
```

Comment:
is it ok to have it also here or should I manually adjust the PRs later in dist-git?

Reply:
I think it's fine.